3-networks-dual-svpc/envs/development/main.tf

```diff
@@ -17,8 +17,8 @@
 locals {
   env              = "development"
   environment_code = substr(local.env, 0, 1)
-  default_region1  = "us-west1"
-  default_region2  = "us-central1"
+  default_region1  = data.terraform_remote_state.bootstrap.outputs.common_config.default_region
+  default_region2  = data.terraform_remote_state.bootstrap.outputs.common_config.default_region_2
```

Comment:
the locals that are populated by the `data.terraform_remote_state` need to be moved to the files `remote.tf` and  `remote.tf.could.example` each with its corresponding version

Reply:
Fixed in bccf82165c7a56a19d559c396b7e551012084d91